Client/src/Components/Inputs/Radio/index.jsx

```diff
@@ -42,6 +42,9 @@ const Radio = (props) => {
 						width: 16,
 						height: 16,
 						boxShadow: `inset 0 0 0 1px ${theme.palette.secondary.main}`,
+						"&:not(.Mui-checked)": {
+							boxShadow: `inset 0 0 0 1px ${theme.palette.text.primary}70`, // Use theme text color for the outline
```

Comment:
Does it make sense to be using the primary text color for a box shadow?  That doesn't seem semantically correct to me.

@marcelluscaio could you comment on this one?  I'll defer to your judement.

It seems to me that just removing the `color: transparent` property of the Radio button should be sufficient here :thinking:

Reply:
Hi @ajhollid , by removing the `color: transparent ` property, it makes the radio button have a  more darker outline than the checkbox. If that is fine then I'll implement it.

Reply:
> Hi @ajhollid , by removing the `color: transparent ` property, it makes the radio button have a more darker outline than the checkbox. If that is fine then I'll implement it.

@Anand-Royy yes I noticed that as well, that's the default appearance of the MUI radio button.

Let's see what @marcelluscaio has to say about the colors, he will be able to clarify RE using a text color here.

@gorkem-bwl should the unselected radio buttons match the color of the checkboxes? Or should they use the default color?

Reply:
> @gorkem-bwl should the unselected radio buttons match the color of the checkboxes? Or should they use the default color?

@ajhollid yes, it should match the color of the checkboxes. 

@Anand-Royy make sure you add a screenshot here as per the PR guidelines.

<img width="523" alt="image" src="https://github.com/user-attachments/assets/0369d6c4-231d-494f-ad6f-6d5c93ac75af" />

Reply:
@gorkem-bwl yes I have added the screenshot just below that under Screenshot section. Is that the correct place to add the screenshot?

Reply:
> @gorkem-bwl yes I have added the screenshot just below that under Screenshot section. Is that the correct place to add the screenshot?

I missed that! Thanks for adding.

Reply:
> Let's see what @marcelluscaio has to say about the colors, he will be able to clarify RE using a text color here.

Waiting for @marcelluscaio

Reply:
Ok we can move forward if we don't hear from him today :)